client/src/main/java/org/apache/falcon/resource/SchedulableEntityInstance.java

```diff
@@ -170,6 +170,11 @@ public int compareTo(SchedulableEntityInstance o) {
             return result;
         }
 
+        result = this.tags.compareTo(o.tags);
+        if (result != 0) {
+            return result;
+        }
+
```

Comment:
Agree with @deepakbarr @pragya-mittal  can you please make the relevant change and update the pull request?